app/components/file-browser/component.ts

```diff
@@ -156,6 +155,20 @@ export default class FileBrowser extends Component.extend({
         return guid ? pathJoin(window.location.origin, guid) : undefined;
     }
 
+    @computed('canEdit', 'loading', 'uploading', 'items', 'filter')
+    get clickable(this: FileBrowser) {
+        const cssClass = ['.dz-upload-button'];
+        if (this.get('loading') || this.get('uploading').length || this.get('filter')) {
+            return cssClass;
+        }
+
+        if (!(this.get('items') && this.get('items').length) && this.get('canEdit')) {
```

Comment:
@hmoco If I'm reading this right... If the user cannot edit these quickfiles, the entire file browser is clickable?

Reply:
Oh, I misparsed parentheses. Looking forward to dropping `.get()` with ember 3.1...

Reply:
me too..

Reply:
In the meanwhile, could be shortened to:
```ts
if (!this.get('items.length') && this.get('canEdit')) {
```
since nested properties will happily resolve to `undefined` if any part of it doesn't exist.